Dockerfile

```diff
@@ -20,7 +20,7 @@ RUN go mod vendor
 RUN go mod tidy
 ARG TEST="true"
 RUN if [ "$TEST" != "false" ]; then ./validate.sh ; fi
-RUN go build -mod=vendor .
+RUN go build -mod=vendor -ldflags "-X github.com/prebid/prebid-cache/version.Ver=`git describe --tags | sed 's/-.*$//'` -X github.com/prebid/prebid-cache/version.Rev=`git rev-parse --short HEAD`" .
```

Comment:
The commands to pull the versions differ from PBS.

Version:
PBS: `git describe --tags | sed 's/^v//'`
PBC: `git describe --tags | sed 's/-.*$//`

Revision:
PBS: `git rev-parse HEAD`
PBC: `git rev-parse --short HEAD`

Are these differences intentional?

Reply:
They are intentional. I'm curious to hear your opinion. The `^v` part only makes sense in Prebid Server because releases of that repo come with a `v` prefix.

**Prebid Server**
```
─➤  git describe --tags
v0.207.0-33-gc8d0e4ab

─➤  git describe --tags | sed 's/^v//'
0.207.0-33-gc8d0e4ab
```

**Prebid Cache**
```
─➤  git describe --tags
0.17.1-10-gb9da543c

─➤  git describe --tags | sed 's/-.*$//'
0.17.1
```

The Prebid Server `version` field includes the release followed by a couple more values which I discarded in the Prebid Cache implementation. Do you think those should be there? If so, we can get rid of the sed command entirely. In the `revision` field value, I included the `--short` flag for the shorter version of the SHA value. 

**Prebid Server**
```
─➤  curl http://localhost:8000/version
{"revision":"c8d0e4ab8ee53eb983d22a287c6ed056af41957c","version":"0.207.0-33-gc8d0e4ab"}
```
**Prebid Cache**
```
─➤  curl http://localhost:2424/version
{"revision":"b9da543c","version":"0.17.1"}
```

Thoughts?